daisy_workflows/build-publish/windows/windows-server-2012-r2-dc-core-uefi.publish.json

```diff
@@ -44,7 +44,7 @@
         "projects/bct-staging-functional/global/licenses/windows-server-2012-r2-dc",
         "projects/bct-staging-functional/global/licenses/windows-server-core"
         {{- else if eq .environment "internal" -}}
-        "projects/windows-internal/global/licenses/internal-windows"
+        "projects/windows-internal/global/licenses/internal-windows",
```

Comment:
The project name is also incorrect. Please update the license url in all the publish files.

The license URL is 
`projects/google.com:windows-internal/global/licenses/internal-windows`

Reply:
Done. Also not sure the service account has access to this project.